src/plugins/saved_objects_management/public/management_section/types.ts

```diff
@@ -6,22 +6,10 @@
  * Side Public License, v 1.
  */
 
-import { SavedObjectReference } from '../../../../core/types';
-
 export interface ObjectField {
   type: FieldType;
   name: string;
   value: any;
 }
 
 export type FieldType = 'text' | 'number' | 'boolean' | 'array' | 'json';
```

Comment:
From a quick grep, I think `ObjectField` and `FieldType` are also no longer used?

Reply:
You're right! This also means we can delete the whole file 😄